backend/tests/services/test_docker_services.py

```
@@ -32,165 +25,76 @@ def mock_env_setup():
-    mock_docker_run.side_effect = side_effect
-
-    result = ensure_services_running()
-    assert result is True
-
-    # Should have tried to start services since they weren't running initially
-    assert any(
-        ("up" in " ".join(call[0][0])) for call in mock_docker_run.call_args_list
+def test_docker_compose_direct_usage(mock_docker_run, mock_env_setup):
+    """Test direct Docker Compose usage without compose_utils wrapper"""
+    # Test direct docker compose command
+    result = subprocess.run(
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)